### PR TITLE
add support for generating push everything policy. 

### PR DESCRIPTION
**Pull request by muralisr**
add flag to evaluate using this policy.

### CODE REVIEW

File: blaze/command/client.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 """ Implements the command for querying a trained policy """
```
Comment (nkansal96) [inline blaze/command/client.py]: File seems to be unnecessarily changed -- should check it out

File: blaze/command/push.py [hunk 2]
```diff
@@ -62,13 +63,22 @@ def random_push_policy(args):
     help="Returns the speed index of the page calculated using pwmetrics. As a float.",
     action="store_true",
 )
+@command.argument(
```
Comment (nkansal96) [inline blaze/command/push.py]: this should be another option in `--policy_type `, not its own option

File: blaze/command/push.py [hunk 3]
```diff
@@ -62,13 +63,22 @@ def random_push_policy(args):
...
+    action="store_true",
+)
 @command.command
 def test_push(args):
     """
     Runs a pre-defined test on the given webpage
     """
-    weight = 0 if args.policy_type == "preload" else 1 if args.policy_type == "push" else None
-    policy_generator = _random_push_preload_policy_generator(weight)
+    if args.use_push_all:
```
Comment (nkansal96) [inline blaze/command/push.py]: should check `policy_type` as mentioned in previous comment

Comment (nkansal96) [reply blaze/command/push.py]: also, shouldn't this condition be negated?

File: blaze/command/push.py [hunk 4]
```diff
@@ -85,6 +95,36 @@ def test_push(args):
     return 0
 
 
+def push_preload_all_policy_generator() -> Callable[[EnvironmentConfig], Policy]:
```
Comment (nkansal96) [inline blaze/command/push.py]: This method signature is not compatible with how the function is used (it expects to be passed a `List[PushGroup]` below). Should change the other generator to also accept an `EnvironmentConfig` and get its push groups from that, and make the changes in `test_push` and `_get_page_load_time_in_replay_server`

Comment (muralisr) [reply blaze/command/push.py]: made params of generator consistent. changing _get_results_in_replay_server, _test_push

**Pull request merged**